File: listas de exercicio/lista 2/lista 2 casa C/questao 7.c
#include <stdio.h>

int main(){
    int N;
    printf("Digite um valor entre 1 a 5 para saber seu respectivo curso:");
    scanf("%d", &N);

    switch (N)
    {
    case 1:
        printf("Engenharia");
        break;
    case 2:
        printf("Edificações");
        break;
    case 3:
        printf("Sistemas Elétricos");
        break;
    case 4:
        printf("Turismo");
        break;
    case 5:
        printf("Análise de Sistemas ");
        break;
    
    default:
    printf("Codigo invalido, insira um valor entre 1 e 5");
        break;
    }
}

File: listas de exercicio/listas 2/lista 2 casa python/questao 7.py
senha = int(input("Digite o codigo de acesso de um curso (1 a 5):"))
if senha == 1:
    print("Engenharia")
if senha == 2:
    print("Edificações")
if senha == 3:
    print("Sistemas Elétricos")
if senha == 4:
    print("Turismo")
if senha == 5:
    print("Análise de Sistemas")
if senha < 1 or senha>5:
    print("erro, digite um valor entre 1 a 5")